pillar/components/firewall.sls

```diff
@@ -10,9 +10,9 @@ firewall:
     services:
       - ssh
       - ftp
-      {%- if salt['cmd.run']('rpm -qa glusterfs-server') %}
+      {#%- if salt['cmd.run']('rpm -qa glusterfs-server') %#}
```

Comment:
This is probably a reason for the cache info output during destroy.

Reply:
will this skip the glusterfs teardown?

Reply:
It won't skip teardown. This is just for firewall port for GlusterFS